src/MonoMod.SourceGen.Internal/Utils/FastDelegateInvokersGenerator.cs

```diff
@@ -79,7 +80,7 @@ private sealed record GeneratorMethod(IMethodSymbol Method, SyntaxTokenList Modi
             ctx.AddSource($"{methodName}.g.cs", sb.ToString());
         }
 
-#pragma warning disable CA1305 // Specify IFormatProvider
+        [SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification="SourceGen is a standalone program, not using specific localization settings for integers isn't that important.")]
```

Comment:
Nit: this isn't strictly true. SourceGen is a Roslyn extension, not a standalone program.

Style nit: The `=` character for the property definition should have spaces around it, and `Justification` should be on a second line because the string is relatively long.